lib/createsend/version.rb

```diff
@@ -1,3 +1,3 @@
 module CreateSend
-  VERSION = "5.1.1" unless defined?(CreateSend::VERSION)
+  VERSION = "5.2.0" unless defined?(CreateSend::VERSION)
```

Comment:
Should we bump the major changes instead (ie: 6.0.0) as it involves a breaking change?